admin/class-acc_user_importer-admin.php

```diff
@@ -435,6 +807,27 @@ private function proccess_user_data ( $users ) {
 					$updated_users_email[] = $userEmail;
 				}
 
+				//Special code is needed to handle a login_name change. wp_update_user()
+				//does not change the login_name or the user_nicename.  This is not
+				//something we want to happen often. But it will happen in the case
+				//where a child record is received before the parent, with the same
+				//email address. Another solution to that would be a pre-processing
+				//step where we re-order the array of incoming registrations,
+				//so that the parent records are received first.
+				if ($loginName != $existingUser->user_login) {
+					$userID = $existingUser->ID;
+					$this->log_dual("> user {$userID} username changed from
+						{$existingUser->user_login} to {$loginName}, update database");
+
+					global $wpdb;
+					$wpdb->update($wpdb->users,
+								  ['user_login' => $loginName],
+								  ['ID' => $existingUser->ID]);
+					$wpdb->update($wpdb->users,
+							      ['user_nicename' => $accUserData['display_name']],
+								  ['ID' => $existingUser->ID]);
+				}
```

Comment:
I can see why you added this, but not sure if that's the best way of handling the scenario. I don't have an idea right now, but I'll have to have a closer look at that whole family membership situation - it is so ugly. I lost eventually track of what they decided, is it exactly one email per family membership, or is it possible that they have one or multiple?